turkce/hatmisekeri.md

```diff
@@ -17,3 +17,4 @@ Bu durumda ne yapardiniz?
 [Duvarları yeme başla?](../turkce/duvarlari-ye/duvarlari-ye.md)
 
 [Duvarları çatalla kaz?](../turkce/catalla-kaz/catalla-kaz.md)
+[Darbe tesebbusu desifre et](../turkce/darbe-kumpas/darbeler.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.